.devcontainer/devcontainer.json

```diff
@@ -2,13 +2,16 @@
 // https://github.com/microsoft/vscode-dev-containers/tree/master/containers/docker-existing-dockerfile
 {
 	"name": "Dev Docker Environment",
-	"image": "osm:dev",
+	// "image": "osm:dev",
```

Comment:
I thought this file isn't even used anymore. Is this needed here or can we remove the file from the repo?

Reply:
This file is needed for local FIPS development because vanilla Go doesn't have the `fipsonly` package. Without using devcontainers, compilation errors will abound

Reply:
I imagine if the file importing that package was conditionally enabled with a build tag then building locally would still work fine when the tag isn't enabled.